### ISSUE TITLE
If modal, keep the focus inside the widget

### ISSUE THREAD

**Issue by dioslaska** [acidb/mobiscroll#290]
The focus should be kept inside the modal popup.

**Comment by dioslaska** [comment]
Done, will be released in 2.15.1
